frontend/src/DmarcReportSummaryGraph.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from 'recharts'
+import { object, number } from 'prop-types'
+import WithPseudoBox from './withPseudoBox'
+import theme from './theme/canada'
+import { Box } from '@chakra-ui/core'
+
+/*
+scheme for const data:
+**strength options: 'strong', 'moderate', 'weak'. Omitted strengths are ignored
+{
+  periods: [
+    {month: STRING, year: INT, property: INT, property: INT, property: INT...},
+    {month: STRING, year: INT, property: INT, property: INT, property: INT...},
+    {...}
+  ],
+  strengths: {
+    strong: {
+      name: "Name to appear to user"
+      types: [
+        "property from periods that are 'strong' ",
+        "property from periods that that are 'strong' ",
+      ]
+    },
+    moderate: {same as strong},
+    weak: {same as strong},
+  }
+}
+*/
+
+function DmarcReportSummaryGraph({ ...props }) {
+  const { data, responsiveWidth } = props
+  const { periods, strengths } = data
+  const ticks = [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1]
+  const { colors } = theme
+
+  const formatTicks = (tick) => {
+    return `${tick * 100}%`
+  }
+
+  // Create date entry for x-axis data keys (e.g.: 'JAN-19')
+  // TODO: Should we just be receiving the data in this format?
```

Comment:
At the time I was not sure how you wanted this so I assumed you wanted the full month name, I can make the change if you would like.